po/zh.po

```diff
@@ -1047,12 +1059,12 @@ msgstr "没有未读的文章"
 #: src/itemlistformaction.cpp:438 src/itemviewformaction.cpp:369
 #: src/view.cpp:915
 msgid "Already on last item."
-msgstr ""
+msgstr "已经位于最后一项。"
 
 #: src/itemlistformaction.cpp:447 src/itemviewformaction.cpp:380
 #: src/view.cpp:881
 msgid "Already on first item."
-msgstr ""
+msgstr "已经位于第一项"
```

Comment:
A full stop is probably missing at the end. The translation immediately before this here uses one, however, the one above that doesn't either. So perhaps you could make this somewhat consistent either way, whatever suits the Chinese language best.

Btw I like how Chinese has its own typographic parentheses, exclamation marks and full stops. :-)